platform/packaging/build/faiss/Dockerfile

```diff
@@ -80,12 +96,12 @@ RUN apt install -y cuda-toolkit-12-2
 RUN apt install -y intel-oneapi-mkl
 RUN apt install -y intel-oneapi-mkl-devel
 
-RUN curl -sLO https://github.com/facebookresearch/faiss/archive/refs/tags/v1.7.4.tar.gz
-RUN tar -xf v1.7.4.tar.gz
+RUN curl -L https://github.com/facebookresearch/faiss/archive/refs/tags/${FAISS_VERSION}.tar.gz -O /workspace/sources/faiss-${FAISS_VERSION}.tar.gz
```

Comment:
The `-O` flag doesn't work for some reason; I was able to replace it with an output redirection.

Reply:
I have changed this in my most recent revision which I will push up shortly, after I can test it builds properly and generates `faiss-1.7.4+0.blah.whl`